app/Policies/RolePolicy.php

```
@@ -0,0 +1,108 @@
+    /**
+     * Determine whether the user can bulk delete.
+     */
+    public function deleteAny(User $user): bool
+    {
+        return $user->can('delete_any_role');
+    }
+
+    /**
+     * Determine whether the user can permanently delete.
+     */
+    public function forceDelete(User $user, Role $role): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$role'. (Unused Code Rules)

(UnusedFormalParameter)